test/v2/unit/coins/btc.js

```diff
@@ -144,25 +147,27 @@ describe('BTC:', function() {
     });
 
     it('should generate 3/3 custom chain p2shP2sh address', function() {
-      const generatedAddress = coin.generateAddress({ keychains, threshold: 3, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH, chain: 20, index: 756 });
+      const addressType = Codes.UnspentTypeTcomb('p2shP2wsh');
+      const chain = Codes.forType(addressType)[Codes.PurposeTcomb('external')];
+      const generatedAddress = coin.generateAddress({ keychains, threshold: 3, addressType, chain, index: 756 });
       const generatedTestAddress = testCoin.generateAddress({
         keychains,
         threshold: 3,
-        addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH,
-        chain: 20,
+        addressType: Codes.UnspentTypeTcomb('p2shP2wsh'),
+        chain,
         index: 756
       });
 
       [generatedAddress, generatedTestAddress].forEach((currentAddress) => {
-        currentAddress.chain.should.equal(20);
+        currentAddress.chain.should.equal(chain);
```

Comment:
How come this test needs changing? Seems like it generated a p2wsh address on chain 20 before. Isn't the updated test doing the same thing?

Reply:
Previously it generated a `p2shP2wsh` address on chain 20, not a `p2wsh` address.